### DISCUSSION TITLE
Unstructured volume hexahedron shape?

### DISCUSSION THREAD

**Discussion by paulmelis** [RenderKit/ospray#563] [Q&A] [answered]
Can hexahedrons be non-rectangular (but still convex)? I’m trying to get a cylindrical volume visualized with an unstructured grid,  but not really sure if that’s supported?

**Comment by johguenther** [comment, marked as answer]
I think so, but better set `hexIterative=true` https://www.ospray.org/documentation.html#unstructured-volume
> hexahedron interpolation method, defaults to fast non-iterative version which could have rendering inaccuracies may appear if hex is not parallelepiped

@gregjohnson?

**Reply by paulmelis** [reply]
Okay, that was a golden tip! I did not see any of the model without that flag, but now at least I can check if I did the grid setup correctly.
